cdk/src/main/java/dev/stratospheric/todoapp/cdk/DomainStack.java

```diff
@@ -55,34 +55,6 @@ public DomainStack(
         .loadBalancerDnsName(networkOutputParameters.getLoadBalancerDnsName())
         .build()
     );
-    ApplicationListener listener = applicationLoadBalancer.addListener(
```

Comment:
I temporarily removed the additional listener (redirect from 80 -> 443), so that the stack doesn't fail upon creation. As part of the current investigation ticket (https://trello.com/c/jd8tyMiQ/244-investigate-how-we-can-improve-our-current-network-ssl-setup), we'll find a different solution

Reply:
I've restored the original state of the stack. The 80 -> 443 redirect is now part of the `NetworkStack` and we're about to refactor the `DomainStack` with subsequent PRs